src/app/share/views/ShareView/ShareFileView.tsx

```diff
@@ -133,8 +134,10 @@ export default function ShareFileView(props: ShareViewProps): JSX.Element {
           name: res.item.plainName,
         });
       })
-      .catch((err) => {
+      .catch(async (err) => {
         if (err.message === 'Forbidden') {
+          const itemData = await shareService.getPublicSharedItemInfo(sharingId);
+          setItemData(itemData);
```

Comment:
Perhaps it is better to extract the logic that makes the call to a function, since the call can give some kind of error and it is convenient to add a try catch as well.